apps/website/src/pages/docs/header.mdx

```diff
@@ -75,7 +56,7 @@ One of the top complaints we received was that our headers visibly takes up too
 
 For accessibility reasons, certain users strictly use their keyboard for navigating digital interfaces. We need to ensure the header doesn’t get in the way of these users every time they want to access a page’s main content. To make their experience more efficient, include a Skip to main content link to allow keyboard users to bypass the navigation header and skip directly to the page’s content.
 
-Without such a link, keyboard users will need to tab through the entire header every time they land on a new page. This makes their experience tedious and time-consuming. For more information, read the Skip to main content link article, and check out the Skip to main content component demo.
+Without such a link, keyboard users will need to tab through the entire header every time they land on a new page. This makes their experience tedious and time-consuming. For more information, read the Skip to main content [article](https://www.freecodecamp.org/news/how-to-add-skip-to-main-content-links-to-a-website/), and check out the Skip to main content [component](https://itwin.github.io/iTwinUI/react/?path=/story/core-skiptocontentlink--basic).
```

Comment:
we should avoid linking to random websites and to storybook as well. I think we might need to add a separate page dedicated to our `SkipToContentLink` component

Reply:
can we use the correct component name (since it already exists and we know its name)?

"Check out the Skip to main content component" → "Check our the `SkipToContentLink` component"

and add an item in #961 for documenting it.